app/controllers/tag_controller.rb

```diff
@@ -470,21 +469,6 @@ def contributors_index
     render template: 'tag/contributors-index'
   end
 
-  def add_parent
```

Comment:
This looks great. We may need to remove the corresponding route: https://github.com/publiclab/plots2/blob/c188b9720ba9cd739321f9efcddbe39810bcce60/config/routes.rb#L209